src/rtext.c

```diff
@@ -1425,40 +1424,24 @@ int TextCopy(char *dst, const char *src)
 // REQUIRES: strcmp()
 bool TextIsEqual(const char *text1, const char *text2)
 {
-    bool result = false;
-
-    if ((text1 != NULL) && (text2 != NULL))
-    {
-        if (strcmp(text1, text2) == 0) result = true;
-    }
-
-    return result;
+    return text1 && text2 ? !strcmp(text1, text2) : 0;
```

Comment:
Direct returns deviates from what's being used as standard structure ([ref1](https://github.com/raysan5/raylib/commit/b68d0850b1dee42d818e67d99f4affe1d890dba1#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R3767-R3777), [ref2](https://github.com/raysan5/raylib/commit/b68d0850b1dee42d818e67d99f4affe1d890dba1#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R3782-R3789), [ref3](https://github.com/raysan5/raylib/commit/b68d0850b1dee42d818e67d99f4affe1d890dba1#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R3808-R3815), [ref4](https://github.com/raysan5/raylib/commit/b68d0850b1dee42d818e67d99f4affe1d890dba1#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R3942-R3947), [etc](https://github.com/raysan5/raylib/commit/b68d0850b1dee42d818e67d99f4affe1d890dba1#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R3964-R3969)).

Reply:
I know this is the coding standard for raylib.  Unless there is also a rule that conditional expressions cannot be used, this leads to some ridiculous results.  E.g.,

```C
unsigned int length = text1 && text2 ? !strcmp(text1, text2) : 0;
return length;
```

(assuming that's a valid initializer, and if not it just gets one more line to declare `length` separately..)

Reply:
Yeah. I rather the current master branch version with `if ((text1 != NULL) && (text2 != NULL))` etc. Since `strcmp()` is still being used, I don't really see any performance gains/reasons for this one.

Reply:
For me, I find

```C
result = false;
if (some-condition) 
   result = true;
```

to be just unnecessary considering that  

``` result = (some-condition);
```

does it directly.   I am dismayed that there is a preference to saying that something that is true is true.   

I would prefer not to encourage the wordier roundabout form.  I accept that I am not the referee here.